src/azure-cli/azure/cli/command_modules/resource/_validators.py

```diff
@@ -109,7 +109,7 @@ def internal_validate_lock_parameters(namespace, resource_group, resource_provid
                                       parent_resource_path, resource_type, resource_name):
     if resource_group is None:
         if resource_name is not None:
-            from msrestazure.tools import parse_resource_id, is_valid_resource_id
+            from azure.mgmt.core.tools import parse_resource_id, is_valid_resource_id
```

Comment:
I think this change is already covered by https://github.com/Azure/azure-cli/pull/29856.